pipelines/respondent-home-ui.yml

```diff
@@ -72,6 +115,9 @@ jobs:
         SURVEY_URL: http://rm-survey-service-concourse-latest.apps.devtest.onsclofo.uk
         SURVEY_USERNAME: ((latest_security_user_name))
         SURVEY_PASSWORD: ((latest_security_user_password))
+        SAMPLE_PASSWORD: firm.wing.weather.legs.opinion.capital
```

Comment:
Should these be `((latest_security_user_password))`?

Reply:
Good spot!

Reply:
Now that's hit github, it should be rotated!

Reply:
I'm not sure if it's that much of a problem as it's latest and no sensitive data lives there. I will check